pkg/v7/chartconfig/desired_test.go

```diff
@@ -30,6 +30,7 @@ func Test_ChartConfig_GetDesiredState(t *testing.T) {
 				Organization: "giantswarm",
 			},
 			expectedChartConfigNames: []string{
+				"coredns-chart",
```

Comment:
This should be after cert-exporter.

Reply:
We talked about the order here https://gigantic.slack.com/archives/C02MS174K/p1537532882000100 at the end we thought that putting coredns (and potentially other fundamental components) first would be better to prevent possible issues (for instance ingress-controller-tempresources requires dns to be in place), let me know WDYT.

Reply:
Ah OK cool. That makes sense. But can you add a comment to the key function?